.cloudbuild/ci/lint.yaml

```diff
@@ -3,6 +3,4 @@ steps:
     id: lint
     args: ['make', 'lint']
 options:
-  pool:
-    name: projects/ci-account/locations/us-west1/workerPools/high-cpu-pool
-
+  machineType: 'E2_HIGHCPU_32'
```

Comment:
Hmm I thought we switched back to shared pool a couple months ago. @tcsc did we just not backport it to v10 back then?

Reply:
It looks like the lint job on master was never changed to the pool (2f1511be88cc76d57368b341796b8b085242bc24), but the pool backport did change the lint job (1b8624dd72fd80ab09b9c07f9d90f432a046ec5c). The pool-revert backport did not remove the pool for the lint job (5514e2d4a239867eee61b3f42fefb43ff51d2b3f).